sapi/fpm/tests/ghsa-54hq-v5wp-fqgv-max-body-parts-default.phpt

```diff
@@ -45,9 +45,9 @@ $tester->close();
 
 ?>
 --EXPECT--
-Warning: PHP Request Startup: Input variables exceeded 20. To increase the limit change max_input_vars in php.ini. in Unknown on line 0
```

Comment:
I think it would be still useful to keep request stage in logs. This was actually added as a feature quite recently.

Reply:
Oh, I see. I just wanted to make them uniform so the code can be reused, but maybe we can detect whether the error happens on startup for all of the errors.